jaeger_client/reporter.py

```diff
@@ -120,14 +122,29 @@ def __init__(self, channel, queue_capacity=100, batch_size=10,
             self.flush_interval = flush_interval or None
             self.io_loop.spawn_callback(self._consume_queue)
 
-        self._process_lock = Lock()
-        self._process = None
+    @staticmethod
+    def _fetch_io_loop(channel, sender):
+        if channel:
+            return channel.io_loop
+        elif sender:
+            return sender.io_loop
+        return None
+
+    def _create_default_sender(self, channel, agent):
```

Comment:
Since `_create_default_sender` is used in only one place why not inline it?

Reply:
Made inline.